xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/internal/html/DefaultHTMLCleaner.java

```diff
@@ -241,9 +241,6 @@ private CleanerProperties getDefaultCleanerProperties(HTMLCleanerConfiguration c
         // See TrimAttributeCleanerTransformation for more information.
         defaultProperties.setTrimAttributeValues(false);
 
-        // This flag should be set to true once https://sourceforge.net/p/htmlcleaner/bugs/221/ is fixed.
-        defaultProperties.setRecognizeUnicodeChars(false);
-
```

Comment:
as `true` is the default value for `recognizeUnicodeChars`, i just scrapped the setter.
Or should I leave it in as an explicit setting with some comment why this is important?

Reply:
IMO I do think it's better that we put the option value explicitely.

Reply:
Ok, agreed. I put it back with an additional comment in the next commit.

Reply:
Yeah, I feel it's safer too, better be explicit and explain why we use this value just in case.

Reply:
Put back setting the value for `recognizeUnicodeChars` in commit with some comment in https://github.com/xwiki/xwiki-commons/pull/149/commits/5cf1562d2cdcd6bad542a333ee8e53ca76ee5505